src/app/main/projects/project-detail/project-detail.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import {projects} from "../all-projects/projects.model";
+import {Router, ActivatedRoute, Params} from "@angular/router";
```

Comment:
use single quotes for imports

import lib dependencies first and after project dependencies - separate them by empty line

```
import { Component, OnInit } from '@angular/core';
import { Router, ActivatedRoute, Params } from '@angular/router';

import { projects } from '../all-projects/projects.model';